library/std/src/gc.rs

```diff
@@ -72,6 +74,29 @@ pub const MIN_ALIGN: usize = 8;
 #[derive(Debug)]
 pub struct GcAllocator;
 
+#[derive(Debug)]
+pub struct GcFinalizedAllocator;
+
+use bdwgc::FinalizerClosure;
+static FINALIZER_CLOSURE: FinalizerClosure =
```

Comment:
Is this a dummy value of sorts? Can we document that.

Reply:
Think of this as the entry point to the finaliser. Boehm's `finalized_malloc` function takes a struct with two fields: a pointer to the fn pointer, and some additional metadata (which we don't use). Since this is re-used for every finalisable object we make this a single static value to save us from having to make a new struct for each object. I'll document this.

Reply:
Right, let's document that then.

Reply:
How about something like this? 8fbd2902e45817e6e156e6843a805f4e024db2c5